action.yml

```
@@ -29,6 +29,9 @@ inputs:
   project:
     description: 'Value passed to the --project flag. The default value is the repository root: "@."'
     default: '@.'
+  annotate:
+    description: 'Whether or not to attempt to create GitHub annotations to show test failures inline. Only effective on Julia 1.8+.'
+    default: 'true'
```

I would suggest that we make this feature opt-in. Otherwise, when Julia 1.8 comes out, people will suddenly start getting these annotations. A lot of people do not like in-line annotations because it makes reviewing PR diffs harder, so having it suddenly be enabled by default will likely give some backlash.

```suggestion
    default: 'false'
```

---

You just press `a` to hide annotations, but I guess most people don't know that

---

TIL. But yeah I've observed multiple users that dislike e.g. Codecov's inline annotations, so I'm very hesistant to turn this on by default.

---

Alright, makes sense. My concern is no one will know about it if it is opt-in, but I guess I can post on discourse or something.

> TIL

FYI you can press `i` to toggler comments when reviewing. Another useful one.